Module not found: Can't resolve './BytesFieldRefInput.schema'

### Bug description

I have model in prisma. Reading, creation objects works fine. Zod schemas generated right.
```
// It works
import { MyModelCreateOneSchema } from "./database/prisma/generated/schemas/createOneMyModel.schema"
```

But when I attempting to import:
```
import { MyModelUpdateOneSchema } from "./database/prisma/generated/schemas/updateOneMyModel.schema"
```
I got error
```
../../packages/database/prisma/generated/schemas/objects/BytesFilter.schema.ts:2:0
Module not found: Can't resolve './BytesFieldRefInput.schema'
  1 | import { z } from "zod"
> 2 | import { BytesFieldRefInputObjectSchema } from "./BytesFieldRefInput.schema"
  3 | import { ListBytesFieldRefInputObjectSchema } from "./ListBytesFieldRefInput.schema"
  4 | import { NestedBytesFilterObjectSchema } from "./NestedBytesFilter.schema"
  5 | 
```

MyModel has relation to other models, one of this using Bytes

### How to reproduce

<!--
1. Go to '...'
2. Change '....'
3. Run '....'
4. See error
-->


### Expected behavior

_No response_

### Prisma information

<!-- Do not include your database credentials when sharing your Prisma schema! -->


### Environment & setup

- OS: <!--[e.g. Mac OS, Windows, Debian, CentOS, ...]-->
- Database: <!--[PostgreSQL, MySQL, MariaDB or SQLite]-->
- Node.js version: <!--[Run `node -v` to see your Node.js version]-->


### Prisma Version

```
4.7.1
```

--- a/prisma/schema.prisma
+++ b/prisma/schema.prisma
@@ -33,6 +33,7 @@ model Post {
   author    User?    @relation(fields: [authorId], references: [id])
   authorId  Int?
   likes     BigInt
+  bytes     Bytes
 }
 
 /// @@Gen.model(hide: true)

--- a/src/transformer.ts
+++ b/src/transformer.ts
@@ -1,12 +1,12 @@
 import type {
   ConnectorType,
-  DMMF as PrismaDMMF,
+  DMMF as PrismaDMMF
 } from '@prisma/generator-helper';
 import path from 'path';
 import {
   checkModelHasModelRelation,
   findModelByName,
-  isMongodbRawOp,
+  isMongodbRawOp
 } from './helpers';
 import { isAggregateInputType } from './helpers/aggregate-helpers';
 import { AggregateOperationSupport, TransformerParams } from './types';
@@ -154,6 +154,10 @@ export default class Transformer {
       } else if (inputType.type === 'True') {
         result.push(
           this.wrapWithZodValidators('z.literal(true)', field, inputType),
+        );
+      } else if (inputType.type === 'Bytes') {
+        result.push(
+          this.wrapWithZodValidators('z.instanceof(Buffer)', field, inputType),
         );
       } else {
         const isEnum = inputType.location === 'enumTypes';

